### PR TITLE
Added github repository hyperlink to feedback strings

### LINKED ISSUE #43262: Add hyperlink to "GitHub repo" text
**This is a Feature Request**

**What would you like to be added**

I want to add a hyperlink to the "GitHub repo".

image

**Comments**

Link to be added: https://github.com/kubernetes/website

### PR DESCRIPTION
**Pull request by Gauravpadam**
Closes #43262

### CODE REVIEW

**Review by satyampsoni — APPROVED** [review]

File: layouts/partials/feedback.html [hunk 1]
```diff
@@ -9,7 +9,7 @@ <h2>{{ T "feedback_heading" }}</h2>
       <a target="_blank" rel="noopener"
         href="https://stackoverflow.com/questions/tagged/kubernetes">
         Stack Overflow</a>.
-      {{ T "layouts_docs_partials_feedback_issue" }}
+      {{ T "layouts_docs_partials_feedback_issue" | safeHTML }}
```
Comment (sftim) [inline layouts/partials/feedback.html:12]: I'd use `markdownify` here, and have the text to localize be Markdown, not raw HTML.

**Review by sftim — COMMENTED** [review]
Thanks

Two things:
- https://www.github.com/Kubernetes/website should be https://www.github.com/kubernetes/website
- I recommend using Markdown not HTML

File: layouts/partials/feedback.html (hunk 1 above)
Comment (Gauravpadam) [reply layouts/partials/feedback.html:12]: Sure, I'll do that

**Review by aj11anuj — COMMENTED** [review]
These changes lgtm, 👍

Here is a suggestion from myside that can be committed in this PR or a new improvement issue can be raised for it.

Screenshot_2023-10-08-15-51-16-398_com android chrome

Here, there are 2 hyperlinks ("Report a problem" and "Suggest an improvement") that is redirecting user to a custom template like below 👇

Screenshot_2023-10-08-15-41-55-079_com android chrome

If a user is new and he files issue with a custom template then he/she/they may likely won't provide all required details that will increase a little burden while addressing that issue. If this custom template link is replaced by proper "Issue template" and "feature template" then it will make our life easier.

Screenshot_2023-10-08-15-43-21-448_com android chrome

**Comment by Gauravpadam** [comment]
@aj11anuj Your suggestion is worth checking out, I'd suggest you raise an issue for it. I can retitle my PR as 'enhancements in footer strings' then ;)

**Pull request merged**